lib/server/index.ts

```diff
@@ -64,7 +66,24 @@ export const main = async (serverOpts: ServerOpts) => {
     getForwardHttpRequestHandler(),
   );
 
-  app.post('/response-data/:brokerToken/:streamingId', handlePostResponse);
+  if (loadedServerOpts.config.BROKER_SERVER_MANDATORY_AUTH_ENABLED) {
+    app.post(
+      '/hidden/brokers/connections/:identifier/auth/refresh',
+      authRefreshHandler,
+    );
+    app.post(
+      '/hidden/brokers/response-data/:brokerToken/:streamingId',
+      handlePostResponse,
+    );
+
+    setInterval(
+      disconnectConnectionsWithStaleCreds,
+      loadedServerOpts.config.STALE_CONNECTIONS_CLEANUP_FREQUENCY ??
+        10 * 60 * 1000,
```

Comment:
enh: would be good to have these defined somewhere as `const`s and imported for re-use/consistency if referenced elsewhere